server/templates/place_landing.html

```diff
@@ -0,0 +1,136 @@
+{#
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+#}
+{%- extends 'base.html' -%}
+
+{% set main_id = 'dc-places-landing' %}
+{% set title = 'Place Explorer' %}
+{% set subpage_title = 'Place Explorer' %}
+{% set subpage_url = url_for('place.place') %}
+
+{% macro place_url(dcid='') -%}
+{{ url_for('place.place', dcid=dcid) }}
+{%- endmacro %}
+
+{% macro ranking_url(sv='', type='', container='') -%}
+{{ url_for('ranking.ranking', stat_var=sv, place_type=type, place_dcid=container, **kwargs) }}
+{%- endmacro %}
+
+{% block head %}
+<link rel="stylesheet"
+  href={{url_for('static', filename='css/place.min.css', t=config['GAE_VERSION'])}}>
+{% endblock %}
+
+{% block content %}
+<div id="body" class="container">
+  <h1 class="mb-4">Place Explorer</h1>
+
+  <div class="search border mb-4">
+    <div id="location-field">
+      <div id="search-icon"></div>
+      <input id="place-autocomplete"
+        placeholder="Enter a country, state, county or city" type="text" />
+    </div>
+  </div>
+
+  <p>The Place Explorer tool helps you understand the core data
```

Comment:
is </p> needed here?

Reply:
done, and closed all the other tags as well